### DISCUSSION TITLE
Is it possible to run multiple integrations with different Amazon accounts

### DISCUSSION THREAD

**Discussion by drick55** [alandtse/alexa_media_player#1323] [Q&A] [answered]
Hi

I would like to use my HA to send TTS announcements to different Alexa devices in my home, however, some of the devices are configured totally separately on different Amazon accounts.

Is it possible to add multiple instances of this integration to HA with different login credentials and linked to different devices?

Thanks

**Comment by alandtse** [comment, marked as answer]
Yes, although you should use include/exclude_devices for any shared devices so they don't show up twice in HA. Check the wiki for info on that configuration.

**Reply by drick55** [reply]
Thanks for the reply, so would I just add additional integrations for each Amazon additional account?

**Reply by alandtse** [reply]
Yes.
